web_src/js/components/DiffFileTreeItem.vue

```diff
@@ -83,10 +83,15 @@ export default {
 span.svg-icon.status {
   float: right;
 }
+
 span.svg-icon.file {
   color: var(--color-secondary-dark-7);
 }
 
+a.file {
+  color: var(--color-text);
+}
```

Comment:
Hmm… That's one approach.
Another would have been to set the `muted` class on the corresponding link in line `13`, which should do the same.